modules/auxiliary/dos/dns/namedown.rb

```diff
@@ -0,0 +1,166 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'timeout'
+require 'socket'
+
+class MetasploitModule < Msf::Auxiliary
+  
+  include Msf::Exploit::Capture
+  include Msf::Auxiliary::UDPScanner
+  include Msf::Auxiliary::Dos
+  include Msf::Auxiliary::Report
+  
+  def initialize(info={})
+    super(update_info(info,
+      'Name'        => 'BIND 9 DoS CVE-2016-2776',
+      'Description' => %q{
+          Denial of Service Bind 9 DNS Server CVE-2016-2776.
+          Critical error condition which can occur when a nameserver is constructing a response.
+          A defect in the rendering of messages into packets can cause named to exit with an
+          assertion failure in buffer.c while constructing a response to a query that meets certain criteria.    
+          
+          This assertion can be triggered even if the apparent source address isnt allowed 
+          to make queries.
+      },
+      # Research and Original PoC - msf module author
+      'Author'      => [ 'Martin Rocha', 'Ezequiel Tavella', 'Alejandro Parodi', 'Infobyte Research Team'], 
+      'License'     => MSF_LICENSE,
+      'References'      =>
+        [
+          [ 'CVE', '2016-2776' ],
+          [ 'URL', 'http://blog.infobytesec.com/2016/09/a-tale-of-packet-cve-2016-2776.html' ]
+        ],
+      'DisclosureDate' => '2016-09-27'
+    ))
+
+    register_options([
+      Opt::RPORT(53),
+      OptAddress.new('SRC_ADDR', [false, 'Source address to spoof'])
+    ])
+  
+    deregister_options('PCAPFILE', 'FILTER', 'SNAPLEN', 'TIMEOUT')
+  end
+
+  def scanner_prescan(batch)
+  	puts '''
```

Comment:
`print_status` or `vprint_status`, but i dont think a big banner like this is needed... cant even tell what it is, a vase?